qsr_lib/src/qsrlib/qsrlib.py

```diff
@@ -88,35 +88,45 @@ class QSRlib(object):
     """The LIB
     """
     def __init__(self, help=True):
-        self.__qsrs = {"rcc2_rectangle_bounding_boxes_2d": QSR_RCC2_Rectangle_Bounding_Boxes_2D(),
-                       "rcc3_rectangle_bounding_boxes_2d": QSR_RCC3_Rectangle_Bounding_Boxes_2D(),
-                       "rcc8_rectangle_bounding_boxes_2d": QSR_RCC8_Rectangle_Bounding_Boxes_2D(),
-                       "cone_direction_bounding_boxes_centroid_2d": QSR_Cone_Direction_Bounding_Boxes_Centroid_2D(),
-                       "qtc_b_simplified": QSR_QTC_B_Simplified(),
-                       "qtc_c_simplified": QSR_QTC_C_Simplified(),
-                       "qtc_bc_simplified": QSR_QTC_BC_Simplified(),
-                       "arg_relations_distance": QSR_Arg_Relations_Distance(),
-                       "moving_or_stationary": QSR_Moving_or_Stationary()}
-
+        # register new qsrs below in qsrs_registry
+        qsrs_registry = (("rcc2", QSR_RCC2_Rectangle_Bounding_Boxes_2D),
+                         ("rcc3", QSR_RCC3_Rectangle_Bounding_Boxes_2D),
+                         ("rcc8", QSR_RCC8_Rectangle_Bounding_Boxes_2D),
+                         ("coneDir", QSR_Cone_Direction_Bounding_Boxes_Centroid_2D),
+                         ("qtcbs", QSR_QTC_B_Simplified),
+                         ("qtccs", QSR_QTC_C_Simplified),
+                         ("qtcbcs", QSR_QTC_BC_Simplified),
+                         ("argd", QSR_Arg_Relations_Distance),
+                         ("mos", QSR_Moving_or_Stationary))
+
+        self.__qsrs = self.check_and_activate_qsrs(qsrs_registry)
```

Comment:
I still don't like that you have to specify the `unique_id` again inside each tuple. If the `unique_id` would be moved out of the init function and therefore made static, you could just do:

```
(QSR_RCC2_Rectangle_Bounding_Boxes_2D.unique_id, QSR_RCC2_Rectangle_Bounding_Boxes_2D)
```

Or even better just have a list of classes and use the `unique_id` in the new `check_and_activate_qsrs` function.